docs/disable_with_comments.rst

```diff
@@ -100,3 +105,19 @@ or:
  key1: value1
  {% endif %}
  key2: value2
+
+Putting template flow control in comments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Alternatively for templating you can wrap the template statements in comments
+to make it a valid YAML file. As long as the templating language doesn't use
+the same comment symbol, it should be a valid template and valid YAML (pre and
+post-template processing).
+
+.. code-block:: yaml
+
+ # This file IS valid YAML because the Jinja is in a YAML comment
+ # {% if extra_info %}
+ key1: value1
+ # {% endif %}
+ key2: value2
```

Comment:
Maybe it would help to have a *good* vs. *bad* example? I let you decide whether to include it or not. Here is an example of how I would write it:

```yaml
 # This file IS NOT valid YAML and will procuce syntax errors
 {% if extra_info %}
 key1: value1
 {% endif %}
 key2: value2
```
vs.
```yaml
 # This file IS valid YAML because the Jinja is in a YAML comment
 # {% if extra_info %}
 key1: value1
 # {% endif %}
 key2: value2
```

Reply:
@adrienverge there's something like what [you're asking for in there already,](https://github.com/adrienverge/yamllint/pull/463/commits/b352c6e3f7119212970c0eecacb4c212ce4895f8#diff-3d7ebf7cfb5f13df97e9cf1fb487910e7ad69145c019f00fbe6f65dd9c286b81R100-R107) it's just been there a while.

Do you think I should rewrite it and put them together more like you did for clarity?

Reply:
Yes maybe the 2 sections would need to be merged somehow. Maybe you could propose something is a new PR, so we can choose the best?

Reply:
@adrienverge sorry for the delay, life got busy. Are you suggesting that I leave this PR open and make a new PR with your suggestion so we can compare them side-by-side, or that I amend this PR?

Reply:
No problem for the delay ;)

> Are you suggesting that I leave this PR open and make a new PR with your suggestion 

Yes exactly, if it's not too much work. I guess it would be easier to compare them side-by-side.